src/components/Home/Home.js

```diff
@@ -27,19 +33,24 @@ function Home() {
                         <CortexCardContents
                             title='Upgraded Engine'
                             body='The source code has been enhanced to improve performance.'
+                            image={DummySquadImg}
+                            alt='some dummies'
```

Comment:
Please change the alt text slightly so it's a little better/more uniform:
`some dummies` -> `Dummy Squad Image`
`a buy menu` -> `Buy Menu Image`
`a joystick` -> `Joystick Image`